requirements/edx/testing.txt

```diff
@@ -101,7 +101,7 @@ edx-api-doc-tools==1.4.0  # via -r requirements/edx/base.txt
 edx-bulk-grades==0.8.1    # via -r requirements/edx/base.txt, staff-graded-xblock
 edx-ccx-keys==1.1.0       # via -r requirements/edx/base.txt
 edx-celeryutils==0.5.2    # via -r requirements/edx/base.txt, super-csv
-edx-completion==3.2.4     # via -r requirements/edx/base.txt
+git+https://github.com/regisb/completion.git@regisb/waffle-override  # via -r requirements/edx/base.txt
```

Comment:
You should be able to restore this to 3.2.4.

Reply:
:ok_hand: